src/command_executor.cc

```diff
@@ -1,29 +1,38 @@
 #include "xcshell/command_executor.h"
 
-#include <unistd.h>
+#include <fcntl.h>
 #include <sys/wait.h>
-#include <csignal>
+#include <unistd.h>
 
+#include <csignal>
 #include <iostream>
 #include <map>
 
+#include "xcshell/CommandParseResult.h"
 #include "xcshell/constants.h"
 #include "xcshell/utils.h"
 
-std::vector<char *> CommandExecutor::BuildArgv(
-    const std::string &command, const std::vector<std::string> &args) {
-  std::vector<char *> argv;
-  argv.reserve(args.size() + 2);
-  argv.push_back(const_cast<char *>(command.c_str()));
-  for (const auto &arg : args) {
-    argv.push_back(const_cast<char *>(arg.c_str()));
+void output_redirect(const CommandParseResult &commandParseResult) {
```

Comment:
Parameter should be snake case, instead of camel case.

Reply:
Can we move this method into class CommandExecutor, instead of global function here?